packages/tooltip/src/useTooltip.ts

```diff
@@ -99,28 +120,39 @@ export const useTooltip = ({
       'aria-describedby': _id,
       'data-garden-container-id': 'containers.tooltip',
       'data-garden-container-version': PACKAGE_VERSION,
+      ref: triggerRef as any,
```

Comment:
Since the user’s `triggerRef` is ultimately forwarded through `getTriggerProps`, would it be possible for `useTooltip` to define its own internal `triggerRef` by default? That way, we could avoid a breaking change.

Reply:
Again, since this concept doesn't align with any of the other containers, I'm not in favor. The idea of a breaking change here is not the problem that we're making it. Containers can easily sustain breaking change updates (as long as the components can absorb them).